feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.ts

```diff
@@ -28,9 +26,10 @@ export class ConfiguratorAttributeHeaderComponent
   iconTypes = ICON_TYPE;
   showRequiredMessageForDomainAttribute$: Observable<boolean>;
 
-  constructor(protected configUtils: ConfiguratorStorefrontUtilsService) {
-    super();
-  }
+  constructor(
+    protected configUtils: ConfiguratorStorefrontUtilsService,
+    protected uiKeyGeneratorService: ConfiguratorUiKeyGeneratorService
```

Comment:
This is breaking change, right?

Reply:
Yes, we will add an annotation for constructor deprecation

Reply:
(Already discussed with @Larisa-Staroverova , but adding in this conversation as well): We should use a more generic name for ConfiguratorUiKeyGeneratorService. What about ConfiguratorAttributeTypeUtilsService?